daemon/serve.cpp

```diff
@@ -310,7 +317,7 @@ int handle_connection(const string &basedir, CompileJob *job,
         if (rmsg.status == 0) {
             write_output_file(obj_file, client);
             if (rmsg.have_dwo_file) {
-                write_output_file(dwo_file, client);
+                write_output_file(dwo_file, client, true /* optional */);
```

Comment:
Please add a comment for why it's optional here, so that one doesn't need to search git history when looking at the code later.

Reply:
+1